lib/fog/compute/kubevirt/models/vm_base.rb

```diff
@@ -68,6 +78,39 @@ def stop(options = {})
           service.update_vm(vm)
         end
       end
+
+      class VmNic
+        attr_accessor :name,
+                      :mac_address,
+                      :type, # values: bridge, slirp
+                      :model,
+                      :ports,
+                      :boot_order
+      end
+
+      class VmNetwork
+        attr_accessor :name,
+                      :type # values: multus, pod, genie
+      end
+
+      class VmDisk
+        attr_accessor :name,
+                      :volume_name,
+                      :type, # values: floppy, disk, lun, cdrom
+                      :image,
+                      :boot_order,
+                      :bus,
+                      :readonly
+      end
+
+      class VmVolume
+        attr_accessor :name,
+                      # values: registryDisk, persistentVolumeClaim, emptyDisk,
```

Comment:
is it intentional to comment `values` here?

Reply:
that's the comment of the supported values for the type - since this one is multiple lines, I had to list it above the 'type' attribute.

Reply:
OK